### PR TITLE
feature/EDU 32 Update user anonymization

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes #31

### CODE REVIEW

File: user-helper.js [hunk 1]
```diff
@@ -1,50 +1,48 @@
...
   return {
-    _id: user._id || uniqueId.create(),
+    _id: uniqueId.create(),
     provider: PROVIDER_NAME_ELMU,
-    username: user.username,
-    passwordHash: await bcrypt.hash(user.password, PASSWORD_SALT_ROUNDS),
-    email: `${user.username}@test.com`,
-    roles: user.roles,
+    username: 'test',
+    passwordHash: await bcrypt.hash('test', PASSWORD_SALT_ROUNDS),
```
Comment (laurentiu-ilici) [inline user-helper.js:13]: I know it's just staging and all, but should we put a slightly better password here? Also, I just realised this is a public repo so anyone can come and see what the admin password is on staging.

Comment (ahelmberger) [reply user-helper.js:13]: Also, anyone can read our comments here ;-)

Comment (adrianaluchian) [reply user-helper.js:13]: It's been like this before. Open project, anonymized data, it's an open playground :)

Comment (ahelmberger) [reply user-helper.js:13]: I created a ticket to create the initial admin user through the app itself, so we don't need to have it here

File: copy-data-from-prod-to-stag [hunk 2]
```diff
@@ -57,16 +57,16 @@ const dumpDir = path.resolve('.tmp/dump');
...
-  const users = await db.collection('users');
+  const usersCollection = await db.collection('users');
 
-  const oldUserIds = (await users.find({}, { projection: { _id: 1 } }).toArray()).map(x => x._id);
-  const newUsers = await userHelper.createTestUsers(oldUserIds);
+  const users = await usersCollection.find({}).toArray();
+  const anonymizeUsers = await userHelper.anonymizeUsers(users);
 
-  await mongoHelper.deleteAllItems(users);
+  await mongoHelper.deleteAllItems(usersCollection);
```
Comment (ahelmberger) [inline copy-data-from-prod-to-stag:65]: Is this actually necessary (we upsert all users later anyway)?

File: user-helper.js (hunk 1 above)
Comment (laurentiu-ilici) [reply user-helper.js:13]: I know they can read our comments :))

**Review by laurentiu-ilici — APPROVED** [review]

File: copy-data-from-prod-to-stag (hunk 2 above)
Comment (laurentiu-ilici) [reply copy-data-from-prod-to-stag:65]: If you really want only the data from prod, you need to. You. might have users that are created only on staging by other means.

Comment (ahelmberger) [reply copy-data-from-prod-to-stag:65]: As far as I can see we re-insert exactly the same set a few lines further down :thinking:

**Review by ahelmberger — APPROVED** [review]

File: copy-data-from-prod-to-stag (hunk 2 above)
Comment (adrianaluchian) [reply copy-data-from-prod-to-stag]: Removed, thank you!

**Pull request merged**